lib/vizkit/vizkit_items.rb

```diff
@@ -803,6 +803,27 @@ def write(&block)
         end
     end
 
+    class TaskModelItem < VizkitItem
+        attr_reader :model
+
+        def initialize(task, options = Hash.new)
```

Comment:
Rather use ruby keyword arguments instead of the old utilrb validate_options. And raise if item_type has an invalid value.

Moreover, I see no need to store the option hash since it's not used anywhere. Or am I missing something ?

~~~
def initialize(task, item_type: :label)
   ...
   if item_type == :value
   elsif item_type == :label
   else
     raise ArgumentError, "#{item_type} is not a valid value for the item_type option. Expected either :label or :value"
   end
~~~